cdci_osa_plugin/osa_lightcurve_query.py

```diff
@@ -377,8 +378,10 @@ def get_data_server_query(self, instrument,
 
         scwlist_assumption, cat, extramodules, inject = OsaDispatcher.get_osa_query_base(
             instrument)
-        E1 = instrument.get_par_by_name('E1_keV').value
-        E2 = instrument.get_par_by_name('E2_keV').value
+
+        E1 = instrument.get_par_by_name('E1_keV', validate=True).value
+        E2 = instrument.get_par_by_name('E2_keV', validate=True).value
```

Comment:
Wasn't the validate input parameter remove in one of the last PRs within the dispatcher?

Reply:
Yes, I removed it, forgot to push!